### DISCUSSION TITLE
Is the "FairyGUI runtime" part of Axmol now?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#1599] [General]
If yes we should add a link like on pugixml.

Latest changes on the official FairyGUI runtime (~2 years old):
![image](https://github.com/axmolengine/axmol/assets/8652787/0e960a8d-8397-4c28-a2c6-b17470d6806e)

And this on the axmol/extension/fairygui 
![image](https://github.com/axmolengine/axmol/assets/8652787/88b91bef-2913-402e-b75e-8e1ec169bc7a)

**Comment by halx99** [comment]
Link to where? all modified of fairygui is in repo

**Reply by aismann** [reply]
see PR #1600

**Comment by aismann** [comment]
PR is written. close it here
